resources/options_lookup.tsv

```diff
@@ -1683,12 +1683,12 @@ Clothes Washer	"EnergyStar, 80% Usage"	ResidentialApplianceClothesWasher	imef=2.
-Clothes Washer	"EnergyStar Most Efficient, 100% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=60.0	annual_cost=6.0	test_date=2013	drum_volume=4.3	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.0	mult_hw=1.0	location=auto	plant_loop=auto																																
-Clothes Washer	"EnergyStar Most Efficient, 120% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=60.0	annual_cost=6.0	test_date=2013	drum_volume=4.3	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.2	mult_hw=1.2	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar More Efficient, 80% Usage"	ResidentialApplianceClothesWasher	imef=2.83	rated_annual_energy=90.0	annual_cost=8.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=0.8	mult_hw=0.8	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar More Efficient, 100% Usage"	ResidentialApplianceClothesWasher	imef=2.83	rated_annual_energy=90.0	annual_cost=8.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.0	mult_hw=1.0	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar More Efficient, 120% Usage"	ResidentialApplianceClothesWasher	imef=2.83	rated_annual_energy=90.0	annual_cost=8.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.2	mult_hw=1.2	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar Most Efficient, 80% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=75.0	annual_cost=7.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=0.8	mult_hw=0.8	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar Most Efficient, 100% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=75.0	annual_cost=7.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.0	mult_hw=1.0	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar Most Efficient, 120% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=75.0	annual_cost=7.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.2	mult_hw=1.2	location=auto	plant_loop=auto																																

```

Comment:
@shorowit How would you feel about adding comments to options_lookup.tsv to document source of option argument values? e.g., 
># https://www.lowes.com/pd/Samsung-AddWash-4-5-cu-ft-High-Efficiency-Stackable-Front-Load-Washer-White-ENERGY-STAR/1000041269

Any better ways to do that?

Reply:
I think the manufacturer and model number is the most appropriate way to do it. You can easily find the specs with those two things, while a link to a store might change.

I think this is a larger issue than just these options though. Given all of the confusion on the appropriate way to create new options for hot water-consuming appliances, I think we need to document all of our options. I vote we go ahead and merge and then deal with the documentation.